ClassicLootManager/Modules/Rosters/RosterManager/Options.lua

```diff
@@ -513,21 +540,44 @@ local function dynamic_item_values(self, roster, equationGet, equationSet, expva
         end),
         name = CLM.L["Exponent / Base"],
         pattern = CONSTANTS.REGEXP_FLOAT,
+        hidden = function()
+            return equationGet() == CONSTANTS.ITEM_VALUE_EQUATION.CUSTOM
+        end
     }
     order = order + 1
-    args["equation_select"] = {
-        type = "select",
-        style = "dropdown",
-        desc = epgpweb .. wowpedia .. equationNote,
+    local variableNames = {}
+    for _, v in pairs(CONSTANTS.CUSTOM_EQUATION_VARIABLES) do
```

Comment:
This is static and really shouldn't be generated dynamically. Create it once outside of options generation.